lib/assessment.md

```diff
@@ -41,7 +52,16 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+```CSS
+.big{
+}
+#contentPane{
+}
+.important.red{
+}
+div p{
```

Comment:
Close. This will grab all `p` elements that are inside a `div` element, _at any level of depth_ - as a result, those elements' parents may not be `div` elements. 

To get a direct child, write `div > p`.